conf/application.conf

```diff
@@ -281,13 +253,6 @@ microservice {
 
 auditing {
   enabled=true
-  traceRequests=true
-  consumer {
```

Comment:
why were the auditing config values removed? they are used here -> https://github.com/hmrc/company-registration-frontend/blob/237137b328d73da00801b5dd4e338ce6a0db3494/it/itutil/IntegrationSpecBase.scala#L52

Reply:
ok sure, have put this back. thanks